src/test/scala/nl/knaw/dans/easy/multideposit/ReadmeSpec.scala

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.multideposit
+
+import java.io.{ByteArrayOutputStream, File}
+
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.scalatest._
+
+class ReadmeSpec extends FlatSpec with Matchers with CustomMatchers {
+  val RES_DIR_STR = new File(getClass.getResource("/").toURI).getAbsolutePath
+
+  val mockedProps = {
+    val ps = new PropertiesConfiguration()
+    ps.setDelimiterParsingDisabled(true)
+    ps.load(new File(RES_DIR_STR + "/debug-config", "application.properties"))
+    ps
+  }
+
+  val mockedArgs = Array("-s", RES_DIR_STR, RES_DIR_STR + "/allfields/input", RES_DIR_STR + "/allfields/output", "datamanager")
+
+  val clo = new ScallopCommandLine(mockedProps, mockedArgs)
```

Comment:
To get rid of the mocking problems add override verify:

```
  private val clo = new CommandLineOptions(Array[String]()) {
    // avoids System.exit() in case of invalid arguments or "--help"
    override def verify(): Unit = {}
  }
```
Should also be done for `easy-stage-dataset` Sorry @paulboon for not seeing/remembering it earlier.